connectors/dify.py

```
@@ -54,14 +57,60 @@ async def send_request_async(self, request_data: Dict[str, Any]) -> Dict[str, An
+        async for line in response.content:
+            line = line.decode('utf-8').strip()
+            if line.startswith('data: '):
+                try:
+                    data = json.loads(line[6:])  # 移除 'data: ' 前缀
+                    event_type = data.get("event")
+                    
+                    if event_type == "message":
+                        message_events.append(data)
+                    elif event_type == "message_end":
+                        message_end_result = data
+                        break  # 找到message_end后停止，这是最完整的响应
```

Ruff (0.12.2)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)